sasmodels/convert.py

```diff
@@ -187,6 +189,16 @@ def _rename_magnetic_pars(pars):
             pars['up_'+k[3:]] = pars.pop(k)
     return pars
 
+
+def _rename_magnetic_angles(pars):
+    """
+    Change name of magnetic angle.
+    """
+    if 'up_angle' in pars:      
+        pars['up_phi'] = pars['up_angle']
```

Comment:
Should this be `'up_theta'` instead of `'up_phi'`?

Also, you will probably want `pars['up_phi'] = 0.` as an additional initializer.

The default `up_theta` is now 90°. Do you need to modify `up_angle` before assigning to `up_theta` (such as adding 90º)? You don't seem to have changed the math, so it is probably okay.

The rest of the code looks fine but I didn't test.

Reply:
up_phi is the angle rotating the polarisation in the scattering plane just like  up_angle did. The new parameter is up_theta that gives the inclination of the magnetisation towards the incoming beam. In the last changes up_theta is now initialised in the function changing the name of magnetic angles.